src/download.rs

```diff
@@ -295,4 +318,346 @@ mod tests {
 
         Ok(())
     }
+    /// Tests that pre message is sent for attachment larger than `PRE_MSG_ATTACHMENT_SIZE_THRESHOLD`
+    /// Also test that pre message is sent first, before the full message
+    /// And that Autocrypt-gossip and selfavatar never go into full-messages
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    async fn test_sending_pre_message() -> Result<()> {
+        let mut tcm = TestContextManager::new();
+        let alice = tcm.alice().await;
+        let bob = tcm.bob().await;
+        let fiona = tcm.fiona().await;
```

Comment:
You can write

```rust
        let alice = &tcm.alice().await;
        let bob = &tcm.bob().await;
        let fiona = &tcm.fiona().await;
```

then you can write `alice` instead of `&alice` everywhere.